modules/support-starting-an-alternative-image-with-toolbox.adoc

```diff
@@ -27,24 +33,19 @@ endif::openshift-origin[]
 # chroot /host
 ----
 
-. Create a `.toolboxrc` file in the home directory for the root user ID:
-+
-[source,terminal]
-----
-# vi ~/.toolboxrc
-----
+. Optional: If you need to use an alternative image instead of the default image, create a `.toolboxrc` file in the home directory for the root user ID, and specify the image metadata:
 +
 [source,text]
 ----
 REGISTRY=quay.io                <1>
 IMAGE=fedora/fedora:33-x86_64   <2>
 TOOLBOX_NAME=toolbox-fedora-33  <3>
```

Comment:
While we're in this document, can we change do the following  since fedora 33 is quite outdated.
```suggestion
IMAGE=fedora/fedora:latest   <2>
```

Reply:
:heavy_check_mark: